src/stc/scintilla/lexers/LexPowerPro.cxx

```diff
@@ -10,7 +10,7 @@
 //
 // Changes:
 // 	2008-10-25 - Initial release
-//	2008-10-26 - Changed how <name> is hilighted in  'function <name>' so that
+//	2008-10-26 - Changed how <name> is highlighted in  'function <name>' so that
```

Comment:
It's not a licensing issue, just a practical one: making changes to the 3rd party sources makes upgrading to their newer versions more difficult as this would result in a merge conflict if this line is modified in Scintilla itself. So it's better not to touch 3rd party sources without a very good reason and fixing this "misspelling" is, frankly, as far from one as it gets...